src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -67,54 +69,128 @@ Lv2SubPluginFeatures::Lv2SubPluginFeatures(Plugin::Type type) :
 
 
 
-void Lv2SubPluginFeatures::fillDescriptionWidget(QWidget *parent,
-													const Key *k) const
+void addHbox(QWidget* parent, const char* left, QString right)
 {
-	const LilvPlugin *plug = getPlugin(*k);
-
-	auto label = new QLabel(parent);
-	label->setText(QWidget::tr("Name: ") + pluginName(plug));
+	if(right.isEmpty()) { return; }
 
-	auto label2 = new QLabel(parent);
-	label2->setText(QWidget::tr("URI: ") +
-		lilv_node_as_uri(lilv_plugin_get_uri(plug)));
-
-	auto maker = new QWidget(parent);
-	auto l = new QHBoxLayout(maker);
+	auto container = new QWidget(parent);
+	auto l = new QHBoxLayout(container);
 	l->setContentsMargins(0, 0, 0, 0);
 	l->setSpacing(0);
 
-	auto maker_label = new QLabel(maker);
-	maker_label->setText(QWidget::tr("Maker: "));
-	maker_label->setAlignment(Qt::AlignTop);
+	auto leftLabel = new QLabel(container);
+	leftLabel->setText(QWidget::tr(left));
+	leftLabel->setAlignment(Qt::AlignTop);
 
-	auto maker_content = new QLabel(maker);
-	maker_content->setText(
-		qStringFromPluginNode(plug, lilv_plugin_get_author_name));
-	maker_content->setWordWrap(true);
+	auto rightLabel = new QLabel(container);
+	if(right.startsWith("http") && !right.contains(' ') && !right.contains('\n'))
+	{
+		right = QString("<a href=\"%1\">%2</a>").arg(right, right);
+		rightLabel->setTextInteractionFlags(rightLabel->textInteractionFlags()
+			| Qt::LinksAccessibleByMouse | Qt::LinksAccessibleByKeyboard);
+		rightLabel->setTextFormat(Qt::RichText);
+		QObject::connect(rightLabel, &QLabel::linkActivated, [](const QString & link){
```

Comment:
Can we have an actual function instead of a lambda, it's less clear this way? Feel free to ignore tho.

Reply:
I usually start making functions if two pieces of code have at least 2 statements (2 lines) in common, but in this case, it is just one line. IMO it looks readable that way.